web/rootfs/defaults/ssl.conf

```diff
@@ -13,10 +13,11 @@ ssl_certificate_key /config/keys/cert.key;
 {{ end }}
 
 # protocols
-ssl_protocols TLSv1.2;
-ssl_prefer_server_ciphers on;
-ssl_ecdh_curve secp384r1;
-ssl_ciphers ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:ECDH+CHACHA20:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES256-SHA:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!3DES:!MD5:!PSK;
+# Mozilla Guideline v5.4, nginx 1.17.7, OpenSSL 1.1.1d, intermediate configuration, no OCSP
+# https://ssl-config.mozilla.org/#server=nginx&version=1.17.7&config=intermediate&openssl=1.1.1d&ocsp=false&guideline=5.4
+ssl_protocols TLSv1.2 TLSv1.3;
```

Comment:
we don't yet support 1.3 because the base image is squeeze, which won't get OpenSSL 1.1.1

Reply:
stretch, that is